insights/tests/client/connection/test_test_connection.py

```diff
@@ -457,23 +448,22 @@ def test_legacy_urls_post_all_fails(get, post, exception, insights_connection):
     get.assert_not_called()
 
 
-@parametrize_exceptions(EXPECTED_EXCEPTIONS_TEST_URLS)
```

Comment:
This was not correct. The `no_catch` tests should have checked for unexpected exceptions.